.lintstagedrc.js

```diff
@@ -8,4 +8,5 @@ const buildEslintCommand = filenames =>
 module.exports = {
   '*.{js,jsx,ts,tsx}': [buildEslintCommand],
   '*': ['yarn pretty:write'], // .prettierrc takes care of filtering files.
+  '*.{js,jsx,ts,tsx}': ['yarn test:all'],
```

Comment:
Good with this for now, but once we get a CI pipeline set up, I think I'd prefer to move this step there to save time on commits.

Reply:
totally agree, just wanted to add for now so tests don't fall by the wayside